db/migrate/20160111111547_change_advocate_persona_type_to_external_user.rb

```diff
@@ -1,9 +1,5 @@
 class ChangeAdvocatePersonaTypeToExternalUser < ActiveRecord::Migration
   def change
-    User.all.each do |user|
-      if user.persona_type == 'Advocate'
-        user.update_column(:persona_type, 'ExternalUser')
-      end
-    end
+    execute(%q{UPDATE users SET persona_type = 'ExternalUser' WHERE persona_type = 'Advocate'} )
```

Comment:
I think this should be an `up` method and a corresponding `down` method needs to be implemented. Otherwise Rails won't be able to reverse this migration.